Content.Server/Chemistry/Components/ReagentDispenserComponent.cs

```diff
@@ -119,6 +123,24 @@ private void InitializeFromPrototype()
             Inventory.Sort(_comparer);
         }
 
+        public void InitializeFromPrototype(string pack)
```

Comment:
this should probably be something like 'addfromprototype' since its not initializing it